src/main/cpp/subsystems/ClimberHAL.cpp

```diff
@@ -0,0 +1,33 @@
+#include "subsystems/ClimberHAL.h"
+
+ClimberHAL::ClimberHAL() {
+    m_climberMotorLeft.RestoreFactoryDefaults();
+    m_climberMotorRight.RestoreFactoryDefaults();
+    
+    error = m_EncoderLeft.SetPositionConversionFactor(wheel_circumference); 
+
```

Comment:
invert left motor based on flag defined in sub system config

Reply:
done